pkg/util/types.go

```diff
@@ -86,6 +86,13 @@ type VolumeInfo struct {
 	JVP string
 }
 
+type LocalHostPathVolInfo struct {
+	VolumeInfo
+	Path string
+	ReclaimPolicy string
+	CasType string
+}
```

Comment:
Please run `gofmt -s -w pkg/util/types.go`, either this tool is not correctly installed on your system or your IDE isn't able to find it.

If you are using vscode, please follow steps from [this StackOverflow answer](https://stackoverflow.com/a/53513867)

Reply:
Okay, I will make sure I run go fmt before every push. Ctrl+s is working but I had autoSave enabled in vscode, so rarely have to explicitly write ctrl+save